src/utils/configLoad.ts

```diff
@@ -102,8 +102,5 @@ export const loadAndValidateConfig = async (
     })
     .catch((error: Error) => {
       onValidationError(error);
-    })
-    .finally(() => {
-      onLoading(false);
```

Comment:
This was causing the loading state to be set to `false` before the services are initialized.

Reply:
Nice find